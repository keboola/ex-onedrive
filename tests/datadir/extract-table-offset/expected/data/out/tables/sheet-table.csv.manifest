{"columns":["Segment","column-2","Country","Duplicate","Duplicate-1","Product","Discount_Band","Units_Sold","column-9","column-10"]}
