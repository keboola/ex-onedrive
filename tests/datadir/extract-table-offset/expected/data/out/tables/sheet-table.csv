"Government","","Canada","","6","Carretera","None","1618.5","",""
"Government","","Germany","","7","Carretera","None","1321","",""
"Midmarket","","France","","8","Carretera","None","2178","","x"
"Midmarket","","Germany","","9","Carretera","None","888","","y"
"Midmarket","(empty header)","Mexico","(duplicate header)","(duplicate header)","Carretera","None","2470","","z"
